mapproxy/Dockerfile

```diff
@@ -21,15 +20,15 @@ RUN apt-get install -y \
     python-lxml \
     libgdal-dev \
     build-essential \
-    python-dev \
+    python3-dev \
     libjpeg-dev \
     zlib1g-dev \
     libfreetype6-dev \
     python-virtualenv
 RUN pip install --upgrade pip
-RUN pip install Shapely Pillow MapProxy==1.8.2 uwsgi
+RUN pip install Shapely Pillow MapProxy==1.8.2 uwsgi requests
 
-EXPOSE 8080
+EXPOSE 8080 1717
```

Comment:
This is also in the docker-compose file.  Might avoid confusion to have in one or the other?